README.md

```diff
@@ -1,29 +1,53 @@
+OpenFisca is a modular project. Depending on your goals, you will install and interact with one or several of the OpenFisca Components.
 
-## What's the purpose?
+[![OpenFisca schema](https://cdn.rawgit.com/openfisca/openfisca-doc/eb7b9ce6f41d7ab2cb7bac08738cfb67d58c0c36/architecture.svg)](https://github.com/openfisca/openfisca-doc/blob/bird_eye_view/architecture.svg)
 
-OpenFisca allows users to calculate many variables of the tax and benefit legislation of a country (social benefits and taxes paid by households) given input variables.
-The impact of reform projects can also be simulated.
+### Web API
+The Web API lets you access the legislation Parameters and Variables.
 
-When plugged on a survey, OpenFisca can also calculate the budgetary consequences of a reform
-and its distributional impact.
+>**Example**: [Mes Aides](https://mes-aides.gouv.fr) uses the OpenFisca Web API to calculate OpenFisca-France benefits.
 
-Its engine is independent of the country, it is therefore possible to simulate any country.
-For now the main supported country is France.
+- To explore the OpenFisca-France Web API services, use the [French Legistation Explorer](https://legislation.openfisca.fr/)
+- To query the Openfisca Web API in your app, see our [Web API endpoints description](https://doc.openfisca.fr/openfisca-web-api/endpoints.html)
+- To host your own instance of the Openfisca API, go to our [installation documentation](https://doc.openfisca.fr/openfisca-web-api/index.html)
+
+### Extensions Packages
+Extensions add on the capacities of a country-package.
+
+>**Example**: See Paris extension and Rennes extension on top of OpenFisca France, Pension extension on top of OpenFisca-Tunisia
+
+- To install an Extension, head to the [Extensions documentation](https://doc.openfisca.fr/contribute/extensions.html)
+- To create a new Extension, head to the [Extension Template on GitHub](https://doc.openfisca.fr/contribute/extensions.html)
+
+### Country package
+Country Packages are the basic modules of OpenFisca. They define the [Parameters](https://doc.openfisca.fr/parameters.html), [Entities](https://doc.openfisca.fr/person,_entities,_role.html) and [Variables](https://doc.openfisca.fr/variables.html) of a country.
 
-OpenFisca is distributed with distinct packages:
+- To install an existing Country-Package, head to the [OpenFisca First Steps page](https://doc.openfisca.fr/first_step.html)
+- To create a new Country Package, head to the [Country Template on GitHub](https://github.com/openfisca/openfisca-extension-template)
 
-- OpenFisca-Core allows to calculate variables in Python
-- OpenFisca-France contains a representation of the French legislation
-- OpenFisca-Web-API allows to calculate variables using an web API
+### OpenFisca Core
+OpenFisca-Core is the main engine: it is the common interface to every Country Package. 
+
+It binds the Web API, Country Package(s), Extension(s) and the engine together. 
```

Comment:
When I read this, I understand that to work with OpenFisca core, I need to provide a Web API, a country package, and maybe an extension. But these three things are not at the same level : the API is packaged in the core.